apps/jenkins/jenkins/jenkins.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           cpu: "2000m"
           memory: "4096Mi"
       javaOpts: >-
-        -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseG1GC -XX:+AlwaysPreTouch -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/jenkins_home/logs -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled -XX:+DisableExplicitGC -Xlog:gc*=info,ref*=debug,ergo*=trace,age*=trace:file=/var/jenkins_home/logs/gc.log::filecount=5,filesize=40M -Djava.util.logging.config.file=/var/jenkins_home/userconf/logging.properties -Djenkins.azure-keyvault.uami.enabled=true -Djenkins.ui.refresh=true -Djenkins.model.Jenkins.logStartupPerformance=true -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.LAUNCH_DIAGNOSTICS=true -Dkubernetes.websocket.ping.interval=60000 -Dorg.apache.commons.jelly.tags.fmt.timeZone=Europe/London -Dhudson.model.WorkspaceCleanupThread.retainForDays=2 -Dhudson.model.DirectoryBrowserSupport.CSP="default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; font-src 'self'; img-src 'self'"
+        -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseG1GC -XX:+AlwaysPreTouch -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/var/jenkins_home/logs -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled -XX:+DisableExplicitGC -Xlog:gc*=info,ref*=debug,ergo*=trace,age*=trace:file=/var/jenkins_home/logs/gc.log::filecount=5,filesize=40M -Djava.util.logging.config.file=/var/jenkins_home/userconf/logging.properties -Djenkins.azure-keyvault.uami.enabled=true -Djenkins.ui.refresh=true -Djenkins.model.Jenkins.logStartupPerformance=true -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.LAUNCH_DIAGNOSTICS=true -Dkubernetes.websocket.ping.interval=60000 -Dorg.apache.commons.jelly.tags.fmt.timeZone=Europe/London -Djava.net.preferIPv4Stack=true -Dhudson.model.WorkspaceCleanupThread.retainForDays=2 -Dhudson.model.DirectoryBrowserSupport.CSP="default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; font-src 'self'; img-src 'self'"
```

Comment:
this affects prod jenkins too, it looks safe but it will cause a restart

Reply:
Ah my bad, missed the `javaOpts` in the other file :)